packages/react/src/themes/teams/components/Dropdown/dropdownStyles.ts

```diff
@@ -31,7 +31,7 @@ const dropdownStyles: ComponentSlotStylesInput<DropdownProps & DropdownState, Dr
     ...(p.toggleIndicator && { paddingRight: v.toggleIndicatorSize }),
   }),
 
-  button: ({ variables: v }): ICSSInJSStyle => {
+  button: ({ props: p, variables: v }): ICSSInJSStyle => {
```

Comment:
this is a bit misleading (although this is not a problem of this PR) - while the name of the slot is `triggerButton`, the names that are used in code varies from `button` (in styles) to `toggleButton` and `triggerButton`. This moment should be made consistent asap

Reply:
changed slot name to `triggerButton`